lib/find.coffee

```diff
@@ -122,6 +122,20 @@ module.exports =
     new Disposable ->
       FileIcons.resetService()
 
+  requestAutocompletions: (editor, labels) ->
+    disposable = @autocompleteWatchEditor?(editor, labels)
+    if disposable?
+      @autocompleteSubscriptions.add(disposable)
+
+  consumeAutocompleteWatchEditor: (watchEditor) ->
+    @autocompleteSubscriptions = new CompositeDisposable
+    @autocompleteWatchEditor = watchEditor
+    if @findView?.findEditor?
+      requestAutocompletions(@findView.findEditor, ['default'])
```

Comment:
Hmm, I think the client libraries (here `find-and-replace`) should choose which providers are applicable, instead of the other way around.

Maybe some more descriptive labels will be helpful later on, such as `active-editor-symbols` or `project-symbols` etc.